### ISSUE TITLE
Remove the expanding boxes in the README

### ISSUE THREAD

**Issue by locallycompact** [IntersectMBO/cardano-addresses#246]
This makes the README completely unsearchable in the browser.

**Comment by abailly** [comment]
I agree, I don't find the folded boxes to be particularly useful. We could provide a _table of contents_ with pointers to various sections already expanded.
